app.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+const express = require('express');
+const path = require('path');
+const nunjucks = require('nunjucks');
+const session =require('express-session')
```

Comment:
This import is unused

Reply:
> This import is unused

which import specifially is unused?

Reply:
`session`

Reply:
login html is being worked on, we just need these changes approved first